components/automate-gateway/api/nodes/nodes.proto

```diff
@@ -142,96 +208,146 @@ service NodesService {
 }
 
 message Id {
+	// UUID for the node.
 	string id = 1;
 }
 
 message Ids {
+	// List of node UUIDs.
 	repeated string ids = 1;
 }
 
 message RerunResponse {}
 
 message Query {
+	// Filters to be applied to the query.
```

Comment:
```suggestion
	// Filters to be applied to the query.
```
Are these the "Available filters" or the "Applied filters"?

Reply:
these are the filters that you want to use to reduce the list down to what you care about